### ISSUE TITLE
Cons (DX11.Geomtry) changes type of resource

### ISSUE THREAD

**Issue by rmtttt** [mrvux/dx11-vvvv#357]
If you make a spread of geometry with cons node, and inputting IndexedGeometry - cons will output spread of IDX11Geometry which cannot be used with IndexIndirect eg.

[ConsGeometry_test.zip](https://github.com/mrvux/dx11-vvvv/files/3216231/ConsGeometry_test.zip)

**Comment by mrvux** [comment]
yes this is correct, since you can cons several geometric types, not only one, havinh more cons is indeed good option, ill need tmake help patches for those tho as it will break the build otherwise
